src/manifolds/MultinomialSymmetric.jl

```diff
@@ -0,0 +1,252 @@
+@doc raw"""
+    MultinomialSymmetric{n} <: AbstractEmbeddedManifold{ℝ, DefaultEmbeddingType}
+
+The multinomial symmetric matrices manifold consists of all matrices with unit sum columns`m` column vectors, where each column is of length
+`n` and unit norm, i.e.
```

Comment:
Are you sure about the unit norm requirement?

Reply:
unit one norm to be precise, I'll rephrase.